addons/compat_rhs_afrf3/compat_rhs_afrf3_overpressure/config.cpp

```diff
@@ -0,0 +1,19 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class SUBADDON {
```

Comment:
really don't think we need these in separate addons
additive configs do no harm to always add